libs/utils/utils.js

```diff
@@ -1293,6 +1281,12 @@ export async function loadArea(area = document) {
     decorateDocumentExtras();
   }
 
+  const icons = area.querySelectorAll('span.icon');
+  if (icons.length) {
+    const { default: decorateIcons } = await import('../features/icons/icons.js');
+    decorateIcons(icons, config);
+  }
```

Comment:
Moving this is something we wanted to avoid in the initial PR as I had discussed with @saugatmalla and @ryanmparrish as it adds a blocking request rather than having it live in parallel with the other ones. We collected real world data on this and want to avoid blocking requests in front of the LCP chain.
![Screenshot 2024-11-13 at 16 58 19](https://github.com/user-attachments/assets/53e4894c-473f-4def-a8c5-086838cf4330)

https://github.com/adobecom/milo/pull/2986#discussion_r1814599321